client/src/components/StatsList/StatButton/StatButton.tsx

```diff
@@ -11,9 +11,12 @@ export const StatButton: React.FC<StatButtonProps> = ({
   handleStatChosen,
   isEnabled,
 }) => {
+  const winningTopTrumpClassName = stat.isTopTrump
```

Comment:
This is poorly named but will leave as is until css help comes along. Will def change before merging.

Reply:
If you remove this:

```const winningTopTrumpClassName = stat.isTopTrump
? "stat-button--top-trump"
    : "stat-button";```

and instead import classnames at the top of the file: `import classnames from "classnames";`
and replace this with:

```const baseClassName = "stat-button";
  const classNames = classnames(baseClassName, {
    [`${baseClassName}--top-trump`]: stat.isTopTrump,
  });```
  
  the button className would then be `className={classNames}`

This will conditionally add the modifier class when the stat is a top trump (see RoundTrackerPip.tsx for another example of this).